src/input/input_reader.rs

```diff
@@ -17,13 +17,11 @@ use crate::action_value::ActionValue;
 pub(crate) struct InputReader<'w, 's> {
     keys: Res<'w, ButtonInput<KeyCode>>,
     mouse_buttons: Res<'w, ButtonInput<MouseButton>>,
-    mouse_motion_events: EventReader<'w, 's, MouseMotion>,
-    mouse_wheel_events: EventReader<'w, 's, MouseWheel>,
+    accumulated_mouse_motion: Res<'w, AccumulatedMouseMotion>,
+    accumulated_mouse_scroll: Res<'w, AccumulatedMouseScroll>,
```

Comment:
Let's make it a bit shorter, we don't have other mouse motion and scroll params anymore:
```suggestion
    mouse_motion: Res<'w, AccumulatedMouseMotion>,
    mouse_scroll: Res<'w, AccumulatedMouseScroll>,
```